.github/workflows/tests.yml

```diff
@@ -18,14 +18,13 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-# pip install -r python/requirements.txt
     - name: Lint with flake8
       run: |
         pip install flake8
         # stop the build if there are Python syntax errors or undefined names
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+        # the GitHub editor is 127 chars wide
+        flake8 . --count --max-complexity=10 --max-line-length=127 --statistics
```

Comment:
by deleting exit-zero, errors are no longer warnings and will return an error message, correct? 
curious why the change in this test file.

Reply:
yes, I want to make the test fail when linting returns errors, so we can follow the lints correctly